cluster/vsphere/util.sh

```diff
@@ -344,7 +344,7 @@ function kube-up {
 
   printf "Waiting for ${KUBE_MASTER} to become available..."
   until curl --insecure --user "${KUBE_USER}:${KUBE_PASSWORD}" --max-time 5 \
-          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta1/pods"; do
+          --fail --output /dev/null --silent "https://${KUBE_MASTER_IP}/api/v1beta3/pods"; do
```

Comment:
+1 for using /healthz or providing a function to test that master is up (We use /healthz in many places already: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/hack/test-cmd.sh#L85).